doc/classes/Vector3.xml

```diff
@@ -270,6 +270,12 @@
 		<member name="x" type="float" setter="" getter="" default="0.0">
 			The vector's X component. Also accessible by using the index position [code][0][/code].
 		</member>
+		<member name="xy" type="Vector2" setter="" getter="" default="Vector2( 0, 0 )">
+			A [Vector2] that represents the vector's X and Y components.
+		</member>
+		<member name="xz" type="Vector2" setter="" getter="" default="Vector2( 0, 0 )">
+			A [Vector2] that represents the vector's X and Z components.
+		</member>
```

Comment:
Why do only these two show up in the docs?

Reply:
There's a reason it's still a draft :)